### DISCUSSION TITLE
First login error

### DISCUSSION THREAD

**Discussion by marvinschm** [nextgenhealthcare/connect#4815] [Q&A]
Hello all,

I am a total newbie.

When I try to connect through the Administrator Launcher I get the following error message: **There was an errror connecting to the server at the specified address. Please verify that the server ip is up and running.**

I logged in with: admin admin

Thanks a lot!

**Comment by pacmano1** [comment]
Same question posted on the forums, see https://forums.mirthproject.io/forum/mirth-connect/support/180100-first-login-error#post180101.   OP, please read the forum thread.

**Reply by michaeljeffreycollins** [reply]
Did the other forum post help you?
